### DISCUSSION TITLE
Auto populate usernames from existing entries

### DISCUSSION THREAD

**Discussion by 0xab3d** [dani-garcia/vaultwarden#4745] [Ideas]
Hello,

Just wanted to ask if there are any plans to add this feature, when you create a new entry, the username as you type it will be auto populated from the existing entries' usernames.

Thanks

**Comment by BlackDex** [comment]
This is probably the wrong place for those questions. 
This project does not maintain the clients. And so we would not know of plans for such a feature.

I would suggest to take a look at the Bitwarden repo's regarding this.

https://community.bitwarden.com/c/feature-requests/5/all

**Reply by 0xab3d** [reply]
Thanks.
